.storybook/main.ts

```diff
@@ -0,0 +1,17 @@
+import type { StorybookConfig } from '@storybook/vue-vite';
+const config: StorybookConfig = {
+  stories: ['../src/**/*.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
+  addons: [
+    '@storybook/addon-links',
+    '@storybook/addon-essentials',
+    '@storybook/addon-interactions'
```

Comment:
What about the new [styling addon](https://storybook.js.org/blog/styling-addon-configure-styles-and-themes-in-storybook/) is there a problem with it?